apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -513,6 +514,39 @@ public long ExecutePartitionedUpdate() =>
         public Task<long> ExecutePartitionedUpdateAsync(CancellationToken cancellationToken = default) =>
             CreateExecutableCommand().ExecutePartitionedUpdateAsync(cancellationToken);
 
+        /// <summary>
+        /// Executes this command as DDL. The command must contain one or more DDL statements;
+        /// <see cref="SpannerConnection.CreateDdlCommand(string, string[])"/> for details.
+        /// </summary>
+        /// <param name="pollUntilCompleted">
+        /// If true, the returned task will wait for the DDL operation to finish execution on Spanner.
+        /// Otherwise, the returned task will finish as soon as the DDL operation has successfully been started on
+        /// Spanner, and the caller can monitor the progress of the long-running operation.
+        /// </param>
+        /// <returns>
+        /// A reference to the long-running operation that was started for the DDL statement(s).
+        /// Note: The operation is null for DropDatabase commands.
+        /// </returns>
+        public Operation ExecuteDdl(bool pollUntilCompleted = false) =>
```

Comment:
Let's keep all the methods in the library async, consumers can always block themselves.